docs/App.Navigation.md

```diff
@@ -129,22 +139,20 @@ Navigation.events().registerAppLaunchedListener(() => {
 
 > You have just set the root using a single stack with the PostsList component AND the Top Bar title provided in the Options object; You can check the complete Options object format [here](https://wix.github.io/react-native-navigation/#/docs/styling?id=options-object-format).
 
-When you refresh the app, you should get the blue PostsList screen:  
+When you refresh the app, you should get the blue PostsList screen:
 
 <img src="https://github.com/wix-playground/wix-mobile-crash-course/blob/master/assets/PostsList.png" align="center" >
 
-All actions described in this section are provided in this [commit](https://github.com/wix-playground/wix-mobile-crash-course/commit/2de9761877a3fa7ed6339daae7d370979da14017).
```

Comment:
@drorbiran good catch. I originally removed it because wasn't sure this really was needed, but as I went on to work on further files, I realized relative commits were mentioned across the course and that they served an important purpose. This should not be deleted, please feel free to revert this change and bring the link back. Thanks!